modules/ui/background.js

```diff
@@ -318,6 +316,45 @@ export function uiBackground(context) {
         }
     }
 
+    function getBackgrounds(filter) {
+        return context.background()
+            .sources(context.map().extent(), context.map().zoom(), true)
+            .filter(filter);
+    }
+
+    function chooseBackgroundAtOffset(offset) {
+        var backgrounds = getBackgrounds(function(d) { return !d.isHidden() && !d.overlay; });
+        backgrounds.sort(sortSources);
+        var currentBackground = context.background().baseLayerSource();
+        var foundIndex = -1;
+        for (let index = 0; index < backgrounds.length; index++) {
+            if (backgrounds[index] === currentBackground) {
+                foundIndex = index;
+                break;
+            }
+        }
```

Comment:
This entire block can be replaced with a call of the array helper method `indexOf()` - it returns -1 if not found: 

`        const foundIndex = backgrounds.indexOf(currentBackground);`

Reply:
Sweet javascripty goodness. Thanks for that. :)